reddit_liveupdate/controllers.py

```diff
@@ -1090,3 +1092,38 @@ def GET_mediaembed(self, liveupdate, embed_index):
         }
 
         return pages.LiveUpdateMediaEmbedBody(**args).render()
+
+
+@add_controller
+class LiveUpdateAdminController(RedditController):
+    @validate(VAdmin())
+    def GET_happening_now(self):
+        candidate_thread_ids = queries.get_active_events()
```

Comment:
I think we can get rid of `candidate_thread_ids` now that we're just using an input, right?